lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -262,17 +262,19 @@ module.exports = class MethodMappings {
             frameId = null;
           }
 
-          await this.driver.manage().switchToFrame({
-            id: frameId
-          });
+          const switchFramePossible = this.driver.wait(until.ableToSwitchToFrame(frameId));
+
+          if (switchFramePossible) {
```

Comment:
this shouldn't have any waiting, it's a simple api mapping. Also, protocol tests should be updated.

Reply:
any updates here?